en/user_guide/media.rst

```diff
@@ -0,0 +1,60 @@
+Media Management
+================
+
+Open Orchestra allows to manage media in the media library panel.
+
+Folder
+------
+
+All files are organised inside folders. It's possible to browse the folder tree to see files by clicking on the left panel.
+
+Create
+~~~~~~
+
+Creating a folder is done by clicking on the "new folder" button in the left panel.
+
+The creation form needs:
+
+* name: name of the folder
+* Websites: which websites are using the folder
+
+Remove
+~~~~~~
+
+To remove a folder, select the folder and click on the trash button on the right panel.
+
+Manage files
+------------
+
+The add button in the folder's content allows to upload files.
+
+A trash icon appears on media preview hovering to delete the media.
+
+Click on media preview open a page with two panels: crop files (for image) and add meta information.
```

Comment:
Rather than "crop files", I'd talk about "format version managment" or something like this as the purpose of this tab is to manage the different versions of the media.